### ISSUE TITLE
Unclean server url handling issue

### ISSUE THREAD

**Issue by DeathRitual** [marklap/rundeckrun#26]
When trying to access the Rundeck API via a URl such as http://rundeckserver:4440/rundeck/api/etc... the requests module throws a HTTPError exception. In our environment the rundeck server resides behind such a URL.

Creating the Rundeck object works fine, but when trying to access Rundeck the Exception rises. Looking into the Rundeck object gives the following:

```python
rd = Rundeck(server='rundeckserver/rundeck', api_token='secret', verify_cert=False)
rd.api.connection.base_api_url
'http://rundeck01.mgmt.dvag.net/rundeck:4440/api'
```

Guess the following should clear the issue at this line: https://github.com/marklap/rundeckrun/blob/master/rundeck/connection.py#L138

```python
        if (protocol == 'http' and port != 80) or (protocol == 'https' and port != 443):
            parted_server = server.partition('/')
            if parted_server[2]:
                self.server = '{0}:{1}/{2}'.format(parted_server[0], port, parted_server[2])
            else:
                self.server = '{0}:{1}'.format(server, port)
```

**Comment by marklap** [comment]
What do you think about this?

https://github.com/marklap/rundeckrun/commit/fab24bc6e02dd6e42f108d1bddb842254d986b36

**Comment by DeathRitual** [comment]
Great!

**Comment by marklap** [comment]
This was merged into 0.2.0
